_episodes/05-loop.md

```diff
@@ -434,9 +416,35 @@ from whatever file is being processed
 > {: . output}
 {: .callout}
 
-Going back to our original file copying problem,
-we can solve it using this loop:
+We would like to modify these files, but also save a version of the original files, naming the copies
```

Comment:
At this point, it's been a while since the basilisk and unicorn files have been mentioned (if you count the 'Spaces in Names' callout - which visually blends in very closely with the main text), so "these files" is a bit of a non-sequitur. Suggested alternative wording:

> Suppose that we plan to modify each of the files in the `data-shell/creatures` directory, but we would first like to save copies of the originals, using the filenames `original-basilisk.dat` and `original-unicorn.dat`.

Reply:
Well spotted!